packages/circuit-ui/components/Table/components/TableCell/TableCell.ts

```diff
@@ -13,13 +13,42 @@
  * limitations under the License.
  */
 
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
+import { FC } from 'react';
 import { css } from '@emotion/core';
 
+import styled, { StyleProps } from '../../../../styles/styled';
+
 const PRESENTATION = 'presentation';
 
-const baseStyles = ({ theme, align }) => css`
+type TableCellProps = {
+  /**
+   * Aligns the content of the Cell with text-align
+   */
+  align?: 'left' | 'right' | 'center';
+  /**
+   * @private Add heading styles to placeholder Cell.
+   * Handled internally
+   */
+  header?: boolean;
+  /**
+   * @private Adds active style to the Cell if it is currently hovered by
+   * sort.
+   * Handled internally
+   */
+  isHovered?: boolean;
+  condensed?: boolean;
+  sortable?: boolean;
+  role?: 'presentation';
+  /**
+   * @private A testid for selecting table cells in tests. Handled internally.
+   */
+  ['data-testid']?: string;
```

Comment:
If you extend the HTML attributes for the `td` element, I think you don't have to specify this prop manually.

Reply:
Looks like I still do unfortunately, I see we're also typing this in `Button` for example:

https://github.com/sumup-oss/circuit-ui/blob/1912119fd998ab9d4000e11db5dfa653fdd8c877/packages/circuit-ui/components/Button/Button.tsx#L75

Reply:
Hm, then my assumption was wrong 🤔